collect_app/src/main/java/org/odk/collect/android/location/client/MaxAccuracyWithinTimeoutLocationClientWrapper.java

```diff
@@ -16,7 +18,7 @@
  *
  * New requests reset the timeout and the highest accuracy.
  */
-public class MaxAccuracyWithinTimeoutLocationClient implements LocationClient.LocationClientListener, LocationListener {
```

Comment:
This name really threw me off, so decided to make it clear this didn't implement `LocationClient`.